waterlog/src/actions/LeakCostsActions.js

```diff
@@ -0,0 +1,15 @@
+import { FETCH_LEAKS_COSTS } from "./types";
+
+export const fetchLeaksCosts = (id = 1) => dispatch => {
+  fetch(`https://localhost:44382/api/segmentleaks/costs/${id}`)
+    .then(res => res.json())
+    .then(costs => {
+      return dispatch({
+        type: FETCH_LEAKS_COSTS,
+        payload: costs
+      });
+    })
+    .catch(function() {
+      console.log("error");
```

Comment:
error tells me its an "error"

Reply:
I have dispatched an action to catch errors

Reply:
File deleted